Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -406,15 +409,14 @@ open class ChartDataSet: ChartBaseDataSet
     {
         calcMinMax(entry: e)
         
-        isIndirectValuesCall = true
```

Comment:
sorry but I have to ask why we can remove it here?

Reply:
Since it's no longer directly accessible, the methods (mostly from collection conformance) call `calcMinMax` only if necessary. No longer need to make an exception for some calls.